Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.30.2"
+version = "0.30.3"
```

Comment:
We don't need to make a release just for this, do we?

Reply:
Would be good to have a release, so that we can bump the downstream candid version.

Reply:
Well, it is a breaking change, so it cannot be a point release as you have marked it. I think it would be better to leave off the release for now, and then we can decide later whether a full version just containing the candid bump is necessary.

Reply:
Okay, sounds good.

Reply:
FYI, I'd like to have a release of the agent in order to upgrade the `ic` repo to candid `0.10` so that could finally make use of that in Internet Identity.